pkg/dao/repository_config_test.go

```diff
@@ -905,43 +906,61 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 		},
 	}
 
-	response, err := dao.ValidateParameters(orgId, parameters[0])
-	require.NoError(t, err)
+	response, err := dao.ValidateParameters(orgId, parameters[0], []string{})
+	assert.NoError(t, err)
+
 	assert.False(t, response.Name.Valid)
 	assert.False(t, response.Name.Skipped)
 	assert.Contains(t, response.Name.Error, "already exists.")
 	assert.False(t, response.URL.Valid)
 	assert.False(t, response.URL.Skipped)
 	assert.Contains(t, response.URL.Error, "already exists.")
 
-	response, err = dao.ValidateParameters(orgId, parameters[1])
-	require.NoError(t, err)
+	response, err = dao.ValidateParameters(orgId, parameters[1], []string{})
+	assert.NoError(t, err)
+
 	assert.False(t, response.Name.Valid)
 	assert.True(t, response.Name.Skipped)
 	assert.False(t, response.URL.Valid)
 	assert.True(t, response.URL.Skipped)
 
-	response, err = dao.ValidateParameters(orgId, parameters[2])
-	require.NoError(t, err)
+	response, err = dao.ValidateParameters(orgId, parameters[2], []string{})
+	assert.NoError(t, err)
+
 	assert.False(t, response.Name.Valid)
 	assert.False(t, response.Name.Skipped)
 	assert.Contains(t, response.Name.Error, "blank")
 	assert.False(t, response.URL.Valid)
 	assert.False(t, response.URL.Skipped)
 	assert.Contains(t, response.URL.Error, "blank")
 
+	mockExtRDao.Mock.On("ValidRepoMD", *parameters[0].URL).Return(200, nil)
+
+	response, err = dao.ValidateParameters(orgId, parameters[0], []string{*parameters[0].UUID})
+	assert.NoError(t, err)
+
+	assert.True(t, response.Name.Valid)
+	assert.False(t, response.Name.Skipped)
+	assert.True(t, response.URL.Valid)
```

Comment:
This is good, can you just move it to be underneath the other parameters[0] test? This way the tests are listed in order.

Reply:
✅